scripts/merge_forced_source_cat.py

```diff
@@ -120,22 +120,9 @@ def load_detector(data_ref, object_table=None, matching_radius=1,
     cat['filter'] = data_ref.dataId['filter']
 
     # Calibrate magnitudes and fluxes
-    calib = data_ref.get('calexp_calib')
-    calib.setThrowOnNegativeFlux(False)
-
-    mag, mag_err = calib.getMagnitude(cat[flux_names['psf_flux']].values,
-                                      cat[flux_names['psf_flux_err']].values)
-
-    cat['mag'] = mag
-    cat['mag_err'] = mag_err
-    cat['SNR'] = np.abs(cat[flux_names['psf_flux']] /
-                        cat[flux_names['psf_flux_err']])
-
-    # TODO Need to calibrate fluxes as well
-    # For now we'll just save this information in an additional column
-    # to be available further downstream.
-    flux_mag0, flux_mag0_err = calib.getFluxMag0()
-    cat['fluxmag0'] = flux_mag0
+    calib = data_ref.get('calexp_photoCalib')
+    # For now we just extract the grey FLUXMAG0
+    cat['FLUXMAG0'] = calib.getInstFluxAtZeroMagnitude()
```

Comment:
lower-case:
`cat['fluxmag0']`

Reply:
I think we should keep the 'mag', 'mag_err', and 'SNR' columns as part of this PR.  
This PR is just about fixing the _calib -> _photoCalib transition.
Removing the 'mag' columns from the catalogs (which we should do eventually) should be done in a coordinated fashion with updates to `gcr-catalogs`.